modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -67,6 +72,10 @@ export default class Ecdsa {
     // Add secret shares
     const x = allShares.map((participant) => BigInt(participant['u'])).reduce(Ecdsa.curve.scalarAdd);
 
+    // Chaincode will be used in future when we add support for key derivation for ecdsa
```

Comment:
Do we have a ticket TODO to link to this?

Reply:
Nope, Key derivation for ecdsa is not in the pipeline as of now, but this is needed to ensure backward compatiblility when it's implemented